airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -708,6 +717,27 @@ private StandardSyncOutput runChildWorkflow(final GeneratedJobInput jobInputs) {
         connectionId);
   }
 
+  private boolean isScheduleToStartTimeout(final Exception e) {
```

Comment:
hacky way to dig through the stack trace to look for a schedule to start timeout, rather than doing `e.getCause().getCause().getCause()` etc. Probably won't do it like this in the real implementation, but wanted something that worked for the spike